ui/src/utils/openUrl.ts

```diff
@@ -4,12 +4,11 @@
  *--------------------------------------------------------------------------------------------*/
 
 // tslint:disable-next-line:no-require-imports
-import opn = require("opn");
+import open = require("open");
 
 export async function openUrl(url: string): Promise<void> {
     // Using this functionality is blocked by https://github.com/Microsoft/vscode/issues/25852:
     // await vscode.commands.executeCommand('vscode.open', vscode.Uri.parse(url));
 
-    // tslint:disable-next-line: no-unsafe-any
-    opn(url);
+    await open(url);
```

Comment:
```suggestion
    void open(url);
```

Should this actually be `void` instead of `await`?

Reply:
I don't think it really matters, but I guess I'd keep it as "await". I could imagine a scenario where we want to wait until the browser is open to continue some sort of UI flow in VS Code